src/components/Pagination/pageButtons.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import getFirstItem from './get-first-item';
+
+export default function PageButtons(props) {
+    const { pages, activePage, onChange } = props;
+
+    const getButtonClassName = (firstItem, index, buttonsToRender) => (
+        classnames(
+            'rainbow-pagination_button',
+            {
```

Comment:
same ^^:
```
classnames('rainbow-pagination_button', {
    'rainbow-pagination_button--first': index === 0,
    'rainbow-pagination_button--last': index === buttonsToRender - 1,
    'rainbow-pagination_button--active': activePage === firstItem + index,
};
```
and if this function does't depend on any prop then move outside the component above